modules/api/src/main/java/io/fluo/api/data/RowColumn.java

```diff
@@ -111,12 +111,12 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return row.hashCode() + col.hashCode();
+    return row.hashCode() + 31 * col.hashCode();
```

Comment:
In commit 71a5f3ec1f5e6ff3f8cf318c79895dba717e275b I also added multiplication by 31 to Column's hashcode function